stl/src/filesystem.cpp

```diff
@@ -850,25 +820,48 @@ struct alignas(long long) _Aligned_file_attrs {
 
     constexpr auto _Get_file_attributes_data =
         __std_fs_stats_flags::_Attributes | __std_fs_stats_flags::_File_size | __std_fs_stats_flags::_Last_write_time;
-    if (_Bitmask_includes(
-            _Flags, _Get_file_attributes_data)) { // caller wants something GetFileAttributesExW might provide
+    if (_Bitmask_includes(_Flags,
+            _Get_file_attributes_data)) { // caller wants something GetFileAttributesExW/FindFirstFileW might provide

```

Comment:
Could you detach the comment so it does not split the line?

Reply:
Background note: These brace-attached comments are relics of our old brace style, when braces appeared on their own lines. When we clang-formatted the codebase in VS 2019 16.0, things like this happened. We generally clean them up by "detaching" comments when working in the area.